tests/integration/test_bundle_deployment.py

```diff
@@ -8,18 +9,21 @@
 from pytest_operator.plugin import OpsTest
 from lightkube.resources.core_v1 import Service
 
-BUNDLE_PATH = "./releases/1.7/edge/kubeflow/bundle.yaml"
 BUNDLE_NAME = "kubeflow"
 
 @pytest.fixture()
 def lightkube_client() -> lightkube.Client:
     client = lightkube.Client(field_manager=BUNDLE_NAME)
     return client
 
+@pytest.fixture
+def bundle_path() -> str:
+    return os.environ.get("BUNDLE_PATH").replace("\"", "")
```

Comment:
Is BUNDLE_PATH passed as a string with `"` in it and that's why we are replacing it here?

Reply:
yes. This is done [here](https://github.com/canonical/bundle-kubeflow/pull/803/files#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449R87-R88). I figured this out by basically running the tests and seeing that without the `replace()` function, the string includes `"` and basically is something like `'\"./releases/1.7/stable/kubeflow/bundle.yaml\"'. Do you want me to add a comment about this?

Reply:
It's fine, I just wanted to understand if it was truly intended that way.